articles.tex

```diff
@@ -192,8 +195,15 @@
 \asubsection{Active Membership Evaluations}
 Active Members are evaluated semi-annually through the Evaluation Process as described in \ref{Evaluations Processes}.
 Failure of the Evaluation Process could result in the member being asked to find alternative housing as soon as possible in accordance with all applicable Residence Life policies regarding room changes.
+\asubsection{Active Membership Leave of Absence}
+An Active Member may at any time request a leave of absence using the process described in \ref{Leave of Absence}. For the duration of an absence, a member:
+\begin{itemize}
+	\item Forfeit their right to vote
+	\item Do not count towards quorum
```

Comment:
Lines 201 and 202 should probably be pluralized. (i.e. Forfeit -> Forfeits and Do -> Does)